scripts/ipintutil

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+
+import netaddr
+import netifaces
+from natsort import natsorted
+from tabulate import tabulate
+
+from sonic_py_common import multi_asic
+from swsssdk import ConfigDBConnector, SonicDBConfig
```

Comment:
swsssdk is deprecated largely in sonic-utilities. Could you adapt the new `ConfigDBConnector` as in latest code?